README.md

```diff
@@ -1,16 +1,21 @@
-<img src="https://cdns.tblsft.com/sites/default/files/blog/hyper_logo_1.jpg" width="100" height="100">
+<img src="https://cdns.tblsft.com/sites/default/files/blog/hyper_logo_1.jpg" width="70" height="70"><img src="https://cdns.tblsft.com/sites/default/files/blog/hyper_logo_1.jpg" width="70" height="70"><img src="https://cdns.tblsft.com/sites/default/files/blog/hyper_logo_1.jpg" width="70" height="70"><img src="https://cdns.tblsft.com/sites/default/files/blog/hyper_logo_1.jpg" width="70" height="70"><img src="https://cdns.tblsft.com/sites/default/files/blog/hyper_logo_1.jpg" width="70" height="70">
 
 
 # Hyper API Samples
-[![Tableau Supported](https://img.shields.io/badge/Support%20Level-Tableau%20Supported-53bd92.svg)](https://www.tableau.com/support-levels-it-and-developer-tools)
+[![Tableau Supported](https://img.shields.io/badge/Support%20Level-Tableau%20Supported-53bd92.svg)](https://www.tableau.com/support-levels-it-and-developer-tools) 
+[![Community Supported](https://img.shields.io/badge/Support%20Level-Community%20Supported-457387.svg)](https://www.tableau.com/support-levels-it-and-developer-tools)
 
 __Current Version:__ 0.0.10622
 
-This repo is the home of the Hyper API sample code. All of the samples are available for each language supported by the Hyper API: Python, Java, C++, and C#/.Net (.NET Standard 2.0). 
+This repo is the home of the Hyper API samples. It contains both __Tableau-Supported Samples__ and __Community-Supported Samples__:
```

Comment:
We should reference the official page for that: https://www.tableau.com/support/itsupport

Reply:
CAn we add a link to the sub-readmes?